lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -214,6 +209,11 @@ const expectations = [
                 description: 'Failed to load resource: the server responded with a status of 404 (Not Found)',
                 url: 'http://localhost:10200/dobetterweb/unknown404.css?delay=200',
               },
+              {
```

Comment:
I wouldn't expect these error messages to change due to this PR (this may be the cause of one of the smoke test failures)